econml/ortho_forest.py

```diff
@@ -690,40 +688,11 @@ def _fit_impl(self, Y, T, X, W=None):
             self.propensity_model, self.model_Y, self.n_T, self.random_state, second_stage=False)
         self.second_stage_nuisance_estimator = DiscreteTreatmentOrthoForest.nuisance_estimator_generator(
             self.propensity_model_final, self.model_Y_final, self.n_T, self.random_state, second_stage=True)
+        self.transformer = FunctionTransformer(func=(lambda T:
```

Comment:
For the ortho forest to have a correct effect, we need to add a one-hot encoder at the end of transform. I suspect that we also need to add the label encoder logic to our dml too. To fix the problem here we could do sth like:
```python
ohe_T = OneHotEncoder(sparse=False, categories='auto').fit(T.reshape(-1, 1))
        self.transformer = FunctionTransformer(func=(lambda T:
                                                    ohe_T.transform(self._label_encoder.transform(self._check_treatment(T)).reshape(-1,1))[:, 1:]),
                                               validate=False)
```

Reply:
Is this something that needs fixing in this PR, or is it covered by #97?

Reply:
no preference. It does depend on this PR, since the structure of the fix will depend on the transformerMixin formulation. But it can be done by @moprescu in a separate PR right after we commit this one.

Reply:
postponing this bug to #97 and to future PR